lib/Constants.ts

```diff
@@ -325,7 +325,7 @@ export const _sensorMap: ISensorMap = {
 		86: 'particulate_matter',
 	},
 }
-export function sensorType(index: number): any {
+export function sensorType(index: number) {
 	const sensorType = {
```

Comment:
There's _probably_ a better type for this which also defines the props, but I don't think it's so important.

Reply:
Type added